package.json

```diff
@@ -93,7 +93,7 @@
     "@angular/ssr": "^17.3.6",
     "@cucumber/cucumber": "^10.8.0",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "7.0.57",
+    "@hmcts/ccd-case-ui-toolkit": "7.0.56-query-management",
```

Comment:
You've gone backwards -ok for now but be careful when you create your final toolkit build